keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/DeliveryConfigRepositoryTests.kt

```diff
@@ -230,6 +247,9 @@ abstract class DeliveryConfigRepositoryTests<T : DeliveryConfigRepository, R : R
                   ),
                   ManualJudgementConstraint()
                 ),
+                verifyWith = setOf(
```

Comment:
It doesn't look like this is being used in the test at all?

Reply:
Line 324

Reply:
Ah, right!